types/index.d.ts

```diff
@@ -2,1342 +2,1454 @@ declare var RA: RamdaAdjunct.Static;
 // TypeScript Version: 2.4
 
 declare namespace RamdaAdjunct {
```

Comment:
You somehow reformatted the entire file and finding out what changed is not impossible. Please return back the formatting and make only necessary changes related to your PR.